backend/src/core/services/credential.service.ts

```diff
@@ -19,6 +20,14 @@ import { UserSettings } from '@core/interfaces'
 const secretsManager = new AWS.SecretsManager(configureEndpoint(config))
 const logger = loggerWithLabel(module)
 
+const twilioCredentialCache = new LruCache<string, string>({
+  max: 1000000, // 1M chars or ~2MB
```

Comment:
Perhaps move these variables into `config` so that we can tune it without code changes.